test/Microsoft.AspNet.Mvc.FunctionalTests/LocalizationTest.cs

```diff
@@ -121,5 +122,36 @@ Learn More
             // Assert
             Assert.Equal(expected, body.Trim());
         }
+
+        [Fact]
+        public async Task Localization_InvalidModel_ValidationAttributes_ReturnsLocalizedErrorMessage()
+        {
+            // Arrange
+            var expected =
+@"<span class=""field-validation-error"" data-valmsg-for=""Name"" data-valmsg-replace=""true"">Nom non valide. Longueur minimale de nom est 4</span>
+<span class=""field-validation-error"" data-valmsg-for=""Product.ProductName"" data-valmsg-replace=""true"">Nom du produit est invalide</span>
+<div class=""editor-label""><label for=""Name"">Name</label></div>
+<div class=""editor-field""><input class=""input-validation-error text-box single-line"" data-val=""true"" data-val-minlength=""Nom non valide. Longueur minimale de nom est 4"" data-val-minlength-min=""4"" id=""Name"" name=""Name"" type=""text"" value=""A"" /> <span class=""field-validation-error"" data-valmsg-for=""Name"" data-valmsg-replace=""true"">Nom non valide. Longueur minimale de nom est 4</span></div>
+
+<div class=""editor-label""><label for=""Product_ProductName"">ProductName</label></div>
+<div class=""editor-field""><input class=""input-validation-error text-box single-line"" data-val=""true"" data-val-required=""Nom du produit est invalide"" id=""Product_ProductName"" name=""Product.ProductName"" type=""text"" value="""" /> <span class=""field-validation-error"" data-valmsg-for=""Product.ProductName"" data-valmsg-replace=""true"">Nom du produit est invalide</span></div>";
+
+            var cultureCookie = "c=fr|uic=fr";
+            var request = new HttpRequestMessage(HttpMethod.Get, "http://localhost/Home/GetInvalidUser");
+            request.Headers.Add(
+                "Cookie",
+                new CookieHeaderValue("ASPNET_CULTURE", cultureCookie).ToString());
+
+            // Act
+            var response = await Client.SendAsync(request);
+            var body = await response.Content.ReadAsStringAsync();
+
+            // Assert
+            // Mono issue - https://github.com/aspnet/External/issues/19
+            Assert.Equal(
+                PlatformNormalizer.NormalizeContent(expected),
```

Comment:
Then it's likely not needed. This thing does a find and replace for certain strings which for which mono doesn't match.